src/uniprotkb/components/column-select/__tests__/ColumnSelectContainer.spec.tsx

```diff
@@ -1,21 +1,15 @@
 import React from 'react';
-import axios from 'axios';
 import { waitFor } from '@testing-library/react';
-import MockAdapter from 'axios-mock-adapter';
 import ColumnSelectContainer, {
   removeFieldFromFieldsData,
 } from '../ColumnSelectContainer';
 import initialState from '../../../../app/state/rootInitialState';
 import renderWithRedux from '../../../../shared/__test-helpers__/RenderWithRedux';
-import mockResultFieldsApi from '../../../__mocks__/resultFieldsData';
-import structuredResultFieldsData from '../../../__mocks__/structuredResultFieldsData.json';
 import { ColumnSelectTab } from '../../../types/resultsTypes';
 import { Column } from '../../../types/columnTypes';
-
-const mock = new MockAdapter(axios);
-mock
-  .onGet(mockResultFieldsApi.request)
-  .reply(200, mockResultFieldsApi.response);
+import structuredResultFieldsData from './__mocks__/structuredResultFieldsData.json';
+import resultFields from '../../../__mocks__/resultFields.json';
+import '../../../__mocks__/mockApi';
```

Comment:
Multiple things here:
 - In general I'm not a big fan of imports without assigning to a variable / side-effect modules
 - we use `axios-mock-adapter` across multiple tests, and with multiple type of responses. I'm not sure if the logic needed for this specific test should be abstracted in a mock file higher in the directory hierarchy, or shoudl be put in a mock file at all.

Reply:
Ah, my bad, I see the same file is indeed used in multiple places. As long as you're putting it only as high in the directory as it is needed I'm fine with it.
Also, you'd need to make sure the mock doesn't interact with other places where we're using a mock for axios (as in importing the mock itself, to be able to call its `restore` method to call it in the `afterAll` jest function)

Reply:
> Ah, my bad, I see the same file is indeed used in multiple places. As long as you're putting it only as high in the directory as it is needed I'm fine with it.

Moved it a bit lower.

> Also, you'd need to make sure the mock doesn't interact with other places where we're using a mock for axios (as in importing the mock itself, to be able to call its restore method to call it in the afterAll jest function)

I've just tested and we can do `mockApi.reset()` if needed. Out of curiosity, when do you think this would be needed in an `afterAll`? Creating a new mock `const mock = new MockAdapter(axios);` in another test file will create a new instance.

Reply:
Well it's a bit of me never entirely trusting how mocks affect each other across tests, so if there's a way to clean them up afterwards I would tend to try to use it.
I don't think it is needed now, as it is working now, but I wouldn't want overlooking that at that stage to cause an unexpected error in the future because of the order current, or new, tests are run.